Description: poppler 0.33 (or something 0.30-0.33) dropped the antialias parameters from GlobalParams
Author: Iain Lane <iain.lane@canonical.com>

--- a/xpdf/xpdf.cc
+++ b/xpdf/xpdf.cc
@@ -175,12 +175,12 @@ int main(int argc, char *argv[]) {
     }
   }
   if (antialiasStr[0]) {
-    if (!globalParams->setAntialias(antialiasStr)) {
+    if (!globalParams->parseYesNo2(antialiasStr, &xpdfParams->antialias)) {
       fprintf(stderr, "Bad '-aa' value on command line\n");
     }
   }
   if (vectorAntialiasStr[0]) {
-    if (!globalParams->setVectorAntialias(vectorAntialiasStr)) {
+    if (!globalParams->parseYesNo2(vectorAntialiasStr, &xpdfParams->vectorAntialias)) {
       fprintf(stderr, "Bad '-aaVector' value on command line\n");
     }
   }
--- a/xpdf/XPDFCore.cc
+++ b/xpdf/XPDFCore.cc
@@ -93,6 +93,10 @@ XPDFCore::XPDFCore(Widget shellA, Widget
 		   GBool installCmap, int rgbCubeSizeA):
   PDFCore(splashModeRGB8, 4, reverseVideoA, paperColorA, !fullScreenA)
 {
+
+  out->setFontAntialias(xpdfParams->antialias);
+  out->setVectorAntialias(xpdfParams->vectorAntialias);
+
   GString *initialZoom;
 
   shell = shellA;
--- a/xpdf/XPDFParams.cc
+++ b/xpdf/XPDFParams.cc
@@ -53,7 +53,7 @@ KeyBinding::~KeyBinding() {
 // XpdfParams
 //------------------------------------------------------------------------
 
-XpdfParams::XpdfParams(char *cfgFileName) {
+XpdfParams::XpdfParams(char *cfgFileName) : antialias(gTrue), vectorAntialias(gTrue) {
   GString *fileName;
   FILE *f;
   int i;
--- a/xpdf/XPDFParams.h
+++ b/xpdf/XPDFParams.h
@@ -88,6 +88,8 @@ public:
   GBool getContinuousView();
   int getPSPaperWidth();
   int getPSPaperHeight();
+  GBool antialias;
+  GBool vectorAntialias;
 
 private:
   void parseCommand(const char *cmdName, GString **val,
